### ISSUE TITLE
Refactor SQL to use prepared queries

### ISSUE THREAD

**Issue by petterroea** [InfectedLan/InfectedAPI#204]

**Comment by halvors** [comment]
Hmm, any specific reason for this? How does this work when using input from PHP?

**Comment by petterroea** [comment]
@halvors security

**Comment by halvors** [comment]
Yeah, but more fuss. But i'm all for it, will prevent SQL injections :)

**Comment by petterroea** [comment]
its like string formatting basically. if anything, i think it will create cleaner SQL queries, the current string appending stuff is pretty ugly, don't you agree?
